stix_shifter_modules/qradar/stix_translation/json/to_stix_map.json

```diff
@@ -829,8 +829,36 @@
       "key": "file.parent_directory_ref",
       "object": "file_target_image",
       "references": "directory_target_image"
+    },
+    {
+      "key": "x-ibm-windows.targetimage",
+      "object": "xwin"
     }
   ],
+  "GrantedAccess": {
+    "key": "x-ibm-windows.granted_access",
```

Comment:
@mdazam1942 I noticed that these `x-ibm-windows` mappings were only in the 2.1 files. I'm not sure if this was just an oversight. Can't think of a reason why they wouldn't be in both unless they were there and then removed at some point.

Reply:
I think I have my answer. https://github.com/opencybersecurityalliance/stix-shifter/commit/f96999cb9517b7313c40a7af90bb9213c8be04ac. Going to fix this.